src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -483,31 +483,15 @@ static void init_transport(grpc_chttp2_transport* t,
              GRPC_CHTTP2_CLIENT_CONNECT_STRLEN);
 
   t->base.vtable = get_vtable();
-  t->ep = ep;
   /* one ref is for destroy */
   gpr_ref_init(&t->refs, 1);
-  t->combiner = grpc_combiner_create();
+  t->ep = ep;
```

Comment:
Could leave a comment somewhere in this functions that these initializations should occur in the same order as the fields are in the struct itself?